AmazonChimeSDK/AmazonChimeSDK/realtime/RealtimeControllerFacade.swift

```diff
@@ -68,4 +68,8 @@ import Foundation
     ///
     /// - Returns: `true` if Amazon Voice Focus is enabled; `false` if Amazon Voice Focus is not enabled, or the audio session was not started yet
     func realtimeIsVoiceFocusEnabled() -> Bool
+
+    func addRealtimeTranscriptEventObserver(observer: TranscriptEventObserver)
```

Comment:
Just curious, can we provide an empty implementation to new methods so that it can avoid to be a breaking change.

Reply:
Did some experiments and default implementation with protocol extension turned out not compatible with `@objc` modifier. But It is possible to declare them optional with @objc modifier like the following:

```
@objc optional func addRealtimeTranscriptEventObserver(observer: TranscriptEventObserver)
```

Will update to make them optional with this approach.